test/solver.jl

```diff
@@ -179,8 +179,8 @@ k = 1
         h_U            = collect(d_U)
         h_Vt           = collect(d_Vt)
         F              = svd(A, full=true)
-        @test abs.(h_U'h_U) ≈ Matrix(one(elty)*I, m, m)
-        @test abs.(h_U[:,1:n]'F.U[:,1:10]) ≈ Matrix(one(elty)*I, n, n)
+        @test abs.(h_U'F.U) ≈ Matrix(one(elty)*I, m, m)
```

Comment:
The old version was correct. The last `m-n` columns of `h_U` need not be orthogonal to the last `m_n` columns of `F.U`.

Reply:
Makes sense.